biz.aQute.bndlib.tests/src/test/AnnotationsTest.java

```diff
@@ -26,6 +26,9 @@
 import aQute.lib.io.IO;
 import junit.framework.TestCase;
 
+@SuppressWarnings({
+		"deprecation", "restriction"
+})
```

Comment:
I had already released commits which fixed these (and the other) warnings. I guess you did not pull from github before you started this. https://github.com/bndtools/bnd/commit/58832868cdc5841bdbdb3be0ecb5e474f2974d72